### PR TITLE
Swagger Update: Software Update deployment  addition (Private Preview)

### PR DESCRIPTION
**Pull request by Atchub**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
Can you please fix the formatting for all documents as there is inconsistencies with indentation.

**Comment by jhendrixMSFT** [comment]
@Atchub has this API been reviewed by ARM and/or the API review board?

File: specification/compute/resource-manager/Microsoft.Compute/preview/2018-11-30-preview/softwareUpdateDeployment.json [hunk 1]
```diff
@@ -0,0 +1,628 @@
...
+			"authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow": "implicit",
+			"description": "Azure Active Directory OAuth2 Flow",
+			"scopes": {
+				"user_impersonation": "impersonate your user account"
+			}
+		}
+	},
+	"paths": {
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/SoftwareUpdateDeployments/{softwareUpdateDeploymentName}": {
```
Comment (jhendrixMSFT) [inline specification/compute/resource-manager/Microsoft.Compute/preview/2018-11-30-preview/softwareUpdateDeployment.json]: Is the leading 'S' in `SoftwareUpdateDeployments` supposed to be capitalized?  I believe this is the reason for [this](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/462070669) failure.

File: specification/compute/resource-manager/Microsoft.Compute/preview/2018-11-30-preview/softwareUpdateDeployment.json [hunk 2]
```diff
@@ -0,0 +1,628 @@
...
+					}
+				},
+				"x-ms-examples": {
+					"Gets a Software Update Deployment.": {
+						"$ref": "./examples/GetSoftwareUpdateDeployment.json"
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/SoftwareUpdateDeployments/{softwareUpdateDeploymentName}/instanceView": {
```
Comment (jhendrixMSFT) [inline specification/compute/resource-manager/Microsoft.Compute/preview/2018-11-30-preview/softwareUpdateDeployment.json]: Missing definition for `softwareUpdateDeploymentName` path parameter.

File: specification/compute/resource-manager/Microsoft.Compute/preview/2018-11-30-preview/softwareUpdateDeployment.json (hunk 1 above)
Comment (jhendrixMSFT) [inline specification/compute/resource-manager/Microsoft.Compute/preview/2018-11-30-preview/softwareUpdateDeployment.json]: Is the leading 'S' in `SoftwareUpdateDeploymentName` supposed to be capitalized?  I believe ARM states this should be camel case (please see other instances of this issue).

File: specification/compute/resource-manager/Microsoft.Compute/preview/2018-11-30-preview/examples/GetSoftwareUpdateDeployment.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{
+	"parameters": {
+		"subscriptionId": "1a7d4044-286c-4acb-969a-96639265bf2e",
+		"resourceGroupName": "mygroup",
+		"vmName": "vmName",
+		"softwareUpdateDeploymentName": "weeklyUpdate",
+		"api-version": "2017-05-15-preview",
```
Comment (safeermohammed) [inline specification/compute/resource-manager/Microsoft.Compute/preview/2018-11-30-preview/examples/GetSoftwareUpdateDeployment.json]: Is the API version correct?

**Comment by jhendrixMSFT** [comment]
@Atchub any update?

**Comment by jhendrixMSFT** [comment]
Closing due to inactivity.  Please re-open when ready to engage.

**Pull request closed without merging**